KWStyleConfigure.h.in

```diff
@@ -11,7 +11,13 @@
      PURPOSE.  See the above copyright notices for more information.
 
 =========================================================================*/
-#define KWSTYLE_MAJOR_VERSION "@KWStyle_MAJOR_VERSION@"
-#define KWSTYLE_MINOR_VERSION "@KWStyle_MINOR_VERSION@"
-#define KWSTYLE_PATCH_VERSION "@KWStyle_PATCH_VERSION@"
+
+#ifndef __KWStyleConfigure_h
+#define __KWStyleConfigure_h
```

Comment:
Could `__` please be removed? Clang flags this with a warning because it is supposed to be reserved for compiler internals.

Reply:
`kwsParser.h` also uses 2 underscores for the "include once macro block" with `__kwsParser_h`. It is the same in `kwsGenerator.h`. I probably just do like in other headers. Is it normal? If no, do you want an other pull/merge request to remove the 2 underscores everywhere?

Reply:
It would be great to remove `__` from the macros in the other headers, too :clap: